scripts/download_munge_wqp.R

```diff
@@ -24,7 +25,15 @@ id_from_status <- function(status.file){
   readLines(status.file, n=1L)
 }
 
-munge_wqp <- function(wqp.data){
-  # dplyr
-  
+munge_secchi <- function(data.in){
+  rename(data.in, Date=ActivityStartDate, value=ResultMeasureValue, units=ResultMeasure.MeasureUnitCode, wqx.id=MonitoringLocationIdentifier) %>% 
+    select(Date, value, units, wqx.id) %>% 
+    mutate(secchi = ifelse(units=='in', value*0.0254, ifelse(units=='ft', value*0.3048, ifelse(units=='cm', value*0.01, value)))) %>% 
+    filter(!is.na(secchi)) %>% 
```

Comment:
Are there going to be more? can you just make a named vector and convert?

TOTALLY untested:
```
unitKey <- setNames(c(0.0254,0.3048,0.04),c("in","ft","cm"))

# then:

mutate(secchi = value * unitKey[units])
```